src/routes/deals.ts

```diff
@@ -7,13 +7,14 @@ import {
   handleGetUprightProfileURL,
   handlePostDeal,
 } from "../controllers/deal";
+import config from "../config";
 
 const deals = {
   name: "routes/deals",
   register: async function (server: Hapi.Server) {
     server.route({
       method: "POST",
-      path: "/webhooks/hubspot/deals",
+      path: `/webhooks/hubspot/deals/${config.dealsHash}`,
```

Comment:
Should we actually use `config.hsHash` here instead? If HubSpot is compromised, then only one hash would be leaked.

Reply:
Yes indeed, I changed it! Well noted Joonas 😊